server/admin_management/api/app/models/model.py

```diff
@@ -233,8 +233,8 @@ class FamAccessControlPrivilege(Base):
         onupdate=datetime.datetime.utcnow,
         comment="The date and time the record was created or last updated.",
     )
-    role = relationship("FamRole", back_populates="fam_access_control_privilege")
-    user = relationship("FamUser", back_populates="fam_access_control_privilege")
+    role = relationship("FamRole", back_populates="fam_access_control_privilege", lazy="joined")
```

Comment:
turns out we have to add `lazy="joined"`, otherwise we can't get role information from the fam_access_control_privilege. By adding it makes the connection.

Reply:
Thanks for verifying that again Catherine!!
Then I guess that's probably the reason `lazy="joined"` was added in other model.
Not important for now, but would be curious how SqlAlchemy can allow to control lazy behaviour.